package.json

```diff
@@ -537,7 +537,7 @@
     "minimatch": "3.0.4",
     "mkdirp": "0.5.1",
     "ms": "2.1.1",
-    "node-environment-flags": "1.0.5",
+    "node-environment-flags": "^1.0.6",
```

Comment:
Please remove the "^", thanks.

Reply:
Perhaps worth configing the `save-prefix` in a project level npmrc so that by default `npm install --save` behaves as intended with project rules?

See https://docs.npm.red/misc/config#save-prefix

Reply:
Updated in 5faba2c3